medcat/meta_cat.py

```diff
@@ -317,8 +368,8 @@ def save(self, save_dir_path: str) -> None:
         # Save tokenizer
         assert self.tokenizer is not None
         self.tokenizer.save(save_dir_path)
-
         # Save config
+
```

Comment:
Let's try and avoid useless whitespace change. It pollutes are changes and makes it more difficult to figure out what was actually changed.